<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>307</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>307</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reversi (Othello)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Users/Vilém/Pictures/Aperture_Science_Logo_by_Zabuza318.jpg</normaloff>../../Users/Vilém/Pictures/Aperture_Science_Logo_by_Zabuza318.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="newGB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Vytvoření hry</string>
    </property>
    <widget class="QPushButton" name="newBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Vytvoří novou hru (C)</string>
     </property>
     <property name="text">
      <string>Nová hra</string>
     </property>
     <property name="shortcut">
      <string>C</string>
     </property>
    </widget>
    <widget class="QComboBox" name="enemyCB">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Výběr typu protihráče</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Člověk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Easy AI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hard AI</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="sizeCB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Výběr velikosti hry</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>6x6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8x8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10x10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12x12</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="sizeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Velikost</string>
     </property>
    </widget>
    <widget class="QLabel" name="enemyLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Protihráč</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="loadGB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Načtení uložené hry</string>
    </property>
    <widget class="QLineEdit" name="loadArg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Zadejte název uložené hry</string>
     </property>
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
    <widget class="QPushButton" name="loadBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Načte uloženou hru (L)</string>
     </property>
     <property name="text">
      <string>Načíst</string>
     </property>
     <property name="shortcut">
      <string>L</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ukončí hru (ESC)</string>
    </property>
    <property name="text">
     <string>Konec</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Otevře nápovědu (F1)</string>
    </property>
    <property name="text">
     <string>Nápověda</string>
    </property>
    <property name="shortcut">
     <string>F1</string>
    </property>
   </widget>
   <zorder>loadGB</zorder>
   <zorder>newGB</zorder>
   <zorder>exitBtn</zorder>
   <zorder>helpBtn</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
